db/projects/iexec-rlc/project.yaml

```diff
@@ -0,0 +1,61 @@
+name: iExec RLC
+alt_names:
+  - iEx.ec
+start_date: '2016-10-16'
+description: 'iExec is building marketplace for cloud resources.'
+web:
+  - 'http://iex.ec/'
+assets:
+  - id: rlc
+    start_date: '2017-01-10'
+    name: iExec RLC
+    type: token
+    token_standard: ERC-20
+    contract_address: '0x607f4c5bb672230e8672085532f7e901544a7375'
+    decimals: 9
+    ledger: ethereum
+    symbol: RLC
+    max_supply: 87000000
+    ico_properties:
+      start_date: '2017-04-19'
+      end_date: '2017-04-19'
+      raised: 12000000 USD
+    webids:
+      coinmarketcap: rlc
+      etherscan/token: RLC
+      cryptocompare: rlc
+      cryptocompare/id: '61877'
+webids:
+  bitcointalk: '1746241.0'
+  custom/chat: 'https://slack.iex.ec/'
+  facebook: iexecteam
+  instagram: iexec_team
+  github: iExecBlockchainComputing
+  linkedin/company: iex.ec
+  medium: iex-ec
+  telegram: joinchat/FswFJw6AaL-kpQeWdBSwRA
+  slack: https://slack.iex.ec/
```

Comment:
Uncorrect. You already have this url in custom/chat.
And also if you write slack: it will give you this result https://https://slack.iex.ec/.slack.com/